database.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$connection = mysqli_connect("localhost", "root", "", "books_db");
+
+if (!$connection) {
```

Comment:
There's no error handling for the database connection. Consider using exceptions to handle connection errors more gracefully and provide more detailed error reporting.

```php
try {
    $connection = new mysqli("localhost", "root", "", "books_db");
    if ($connection->connect_error) {
        throw new Exception("Connection failed: " . $connection->connect_error);
    }
} catch (Exception $e) {
    echo "Error: " . $e->getMessage();
    exit;
}
```